packages/signal-persister/src/services/storeSignal.service.ts

```diff
@@ -28,7 +28,9 @@ export function storeSignalServiceBuilder(db: DB) {
 
         /* it means that signal is already present on db */
         if (signalRecordId !== null) {
```

Comment:
I propose to remove the comments and write down:

(see https://critter.blog/2020/09/15/dont-comment-your-code-refactor-it/ Martin Fowler citation)


```
if (signalIsAlreadyPresent(signalRecordId)) {
...
}

// outside module
function signalIsAlreadyPresent(signalRecordId: number | null): boolean {
  return signalRecordId !== null;
}
```

Reply:
Good point!